### ISSUE TITLE
Handle translated fields when sending metadata to Cinder

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#21542]
See also metadata issue: https://github.com/mozilla/addons-server/issues/21234

Add-ons & collections can contain data in multiple locales. Latest proposal is to send the data in the reporter's locale, but with Cinder merging data for a given entity, that seems problematic, if 2 reports are sent from reporters using different locales the Cinder moderator would only see the content in the locale of the last reporter, unless they dig into the entity's history (and assuming that history is available).

Investigate whether or not we can send the translated data using the Dictionary type in Cinder, and how merging would work in that scenario.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/ADDSRV-606)

**Comment by diox** [comment]
Unfortunately Cinder doesn't merge dictionary contents, if I send the following (in this order):
- `"name_translations": {"fr": "Something in FR"}`
- `"name_translations": {"en-US": "Something in en-US"}`

Then when looking at the job in Cinder we only see the FR variant before the en-US report is sent. After, we only see `"name_translations": {"en-US": "Something in en-US"}`.

**Comment by diox** [comment]
> Latest proposal is to send the data in the reporter's locale, but with Cinder merging data for a given entity, that seems problematic, if 2 reports are sent from reporters using different locales the Cinder moderator would only see the content in the locale of the last reporter, unless they dig into the entity's history (and assuming that history is available).

History is available, and that's the proposed solution from Cinder.

**Comment by ioanarusiczki** [comment]
@diox 

This is how I tested:

 - from AMO I went to the detail page of an add-on with metadata in multiple locales and used some of those locales in the reporting form
 https://addons-dev.allizom.org/de/firefox/feedback/addon/ad-blocker_localesforcinder/ or
 https://addons-dev.allizom.org/fr/firefox/feedback/addon/ad-blocker_localesforcinder/

Went to Cinder and checked the content of the report and I saw that the content is translated in the locale I used when I reported the add-on. I've noticed that even the promoted badge has a translation.

- I made multiple reports using various locales. Cinder displays the content of the last reported locale. The other reports are displayed in Relationships, each has in the Summary the Locale in which it's been reported. 

- from addon manager I've reported extensions after changing in the reporting form's url en-US with other locales. Same results as above.

If there's anything else I should test let me know.
